etc/config/components/kfp_component_catalog.json

```diff
@@ -3,19 +3,19 @@
     "run-notebook-using-papermill": {
       "name": "Run Notebook Using Papermill",
       "path": {
-        "url": "https://raw.githubusercontent.com/kubeflow/pipelines/1.4.1/components/notebooks/Run_notebook_using_papermill/component.yaml"
+        "filename": "resources/kfp/run_notebook_using_papermill/component.yaml"
```

Comment:
I would suggest we move the `resources/kfp` portion of these into the server code that, today, uses `__file__` in #1801.  That way, _when_ we move these files out of our installed location, we only need to update the server code that concatenates to `__file__` and don't have to change both the server and the catalog file entries themselves.

Reply:
I agree